bridge/github/import.go

```diff
@@ -87,16 +72,17 @@ func (gi *githubImporter) ImportAll(repo *cache.RepoCache, since time.Time) erro
 		}
 	}
 
-	if err := iterator.Error(); err != nil {
+	if err := gi.iterator.Error(); err != nil {
 		fmt.Printf("import error: %v\n", err)
 		return err
 	}
 
-	fmt.Printf("Successfully imported %v issues from Github\n", iterator.ImportedIssues())
+	fmt.Printf("Successfully imported %d issues from Github\n", gi.importedIssues)
```

Comment:
What about a unified way to phrase that ? `Successfully imported %d issues and %s identities from Github` ?